mount/mount_windows.go

```diff
@@ -55,6 +55,46 @@ func (mounter *Mounter) Mount(source string, target string, fstype string, optio
 	return mounter.MountSensitive(source, target, fstype, options, nil /* sensitiveOptions */)
 }
 
+//mountNfs uses powershell command "net use" to connect NFS share
+func mountNfs(source, target string, options []string) error {
+	mountCmd := "powershell /c net use " + source
+	output, err := exec.Command(mountCmd, options...).CombinedOutput()
+	if err != nil {
+		klog.Errorf("NFS mount failed: %v, source(%q) output: %q", err, source, target, string(output))
+		return err
+	}
+
+	klog.V(4).Infof("NFS mount source(%q) on target(%q) successfully, output: %q", source, target, string(output))
+	return nil
+}
+
+//mountSmb tries to create SMB mapping to a SMB share
+func mountSmb(source, target string, options []string) error {
+	if len(options) < 2 {
+		klog.Warningf("mount options(%q) command number(%d) less than 2, source:%q, target:%q, skip mounting",
+			sanitizedOptionsForLogging, len(options), source, target)
+		return nil
```

Comment:
This isn't your change, but is skipping mounting the right thing to do?

Reply:
I feel it should fail the operation. @andyzhangx is there some reason for skipping here?

Reply:
I think we can return error here if `len(options) < 2 `

Reply:
will create another PR for it.